src/client/renderer/r_context.c

```diff
@@ -40,6 +40,19 @@ static void R_SetWindowIcon(void) {
 }
 
 /**
+ * @brief Attempts to create the specified versioned context, returns true on success and false on failure.
+ */
+static _Bool R_InitGLContext(const int major, const int minor) {
+
+	SDL_GL_SetAttribute(SDL_GL_CONTEXT_MAJOR_VERSION, major);
+	SDL_GL_SetAttribute(SDL_GL_CONTEXT_MINOR_VERSION, minor);
+		
+	return (r_context.context = SDL_GL_CreateContext(r_context.window)) != NULL;
+}
```

Comment:
This helper seems kind of superfluous now that we're only trying one GL version.